packages/ui/v2/core/layouts/SettingsLayout.tsx

```diff
@@ -120,44 +120,46 @@ const SettingsSidebarContainer = ({ className = "" }) => {
         <div className="desktop-only pt-4" />
         <div>
           <VerticalTabItem
-            name="Settings"
+            name="Back"
             href="/"
             icon={Icon.FiArrowLeft}
             textClassNames="text-md font-medium leading-none text-black"
+            className="my-6"
           />
         </div>
         {tabsWithPermissions.map((tab) => {
           return tab.name !== "teams" ? (
             <React.Fragment key={tab.href}>
-              <div>
-                <div className="group flex h-9 w-64 flex-row items-center rounded-md px-3 py-[10px] text-sm font-medium leading-none text-gray-600 hover:bg-gray-100  group-hover:text-gray-700 [&[aria-current='page']]:bg-gray-200 [&[aria-current='page']]:text-gray-900">
+              <div className={`${!tab.children?.length ? "!mb-3" : ""}`}>
```

Comment:
Overall looks good - great job. However, I can't figure out this line is doing -> I've tried removing it and have noticed no difference.

Reply:
For example, if there are no children in the tab, we would add the "mb-3" class, but if there are some children, we don't want the extra margin. The same logic applies below

Reply:
Okay sweet - is there any reason it is set as important? There is nothing else assigning mb on this div so I don't believe it is needed

Reply:
Actually i tried removing it, but it doesn't work that way (not sure about the reason)